ajax/libs/webtorrent-component/package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Would you please remove this file and `Add webtorrent-component w/ npm auto-update via single package.json` this commit?
In this PR, we don't need to add this file and the second commit.

Many thanks.